pkg/eval/mods/file/file.go

```diff
@@ -0,0 +1,48 @@
+// File module which is an equivalent of POSIX like commands.
+// Checkout issule number 1263 for more information.
+
+package file
+
+import (
+	"os"
+
+	"src.elv.sh/pkg/eval"
+	"src.elv.sh/pkg/eval/vals"
+)
+
+// elvdoc:fn fopen
+// ```elvish
+// fopen $filename
+// ```
+// Opens the file for reading.
+// File should be closed after opening using fclose
+// Example
+//  ```elvish-transcript
+// ~> cat a.txt
+// Some text in file
+// f = (fopen a.txt)
+// ~> cat < $f
+// Some text in file
+// ~> fclose $f
+// ```
```

Comment:
Elvdocs are markdown after `//`, so individual paragraphs should be separated by lines containing just `//`.

To build the reference doc for the new file module, add it to website/ref/index.toml and write a prelude Markdown file; see website/ref/math.toml for an example.

Reply:
Also, note that the text needs to be updated to reflect the new name of the command; i.e., `file:close` rather than `fopen`.